pkg/asset/installconfig/aws/availabilityzones.go

```diff
@@ -28,9 +29,18 @@ func availabilityZones(ctx context.Context, session *session.Session, region str
 		return nil, errors.Wrap(err, "fetching availability zones")
 	}
 
+	// This regex finds local zones and allows us to filter them out.
+	// This is a temporary fix until we update our aws-sdk-go package to a
+	// version that adds the ZoneType field to the AvailabilityZone struct
+	regex, err := regexp.Compile("^\\w+-\\w+-\\w+-\\w+-\\w+$")
```

Comment:
I find it a bit easier to read without needed to have the escapes for the backslash.
```suggestion
	regex, err := regexp.Compile(`^\w+-\w+-\w+-\w+-\w+$`)
```
I would also make this a package-level variable and use `MustCompile` instead. The regex is static so we know it must compile (or it is a coding error).

However, a regex seems like it may be overkill here. All we need to do is search for hyphens in the string. I'm not sure which people think is easier to understand. They both need examples, either way.
```
len(strings.Split(zoneName, "-")) == 5
```